packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/eventbridge-scheduler/create-schedule.ts

```diff
@@ -0,0 +1,476 @@
+import { Construct } from 'constructs';
+import * as iam from '../../../aws-iam';
+import * as kms from '../../../aws-kms';
+import * as sqs from '../../../aws-sqs';
+import * as sfn from '../../../aws-stepfunctions';
+import { Stack, Duration, Token } from '../../../core';
+import { integrationResourceArn } from '../private/task-utils';
+
+/**
+ * Schedule for EventBridge Scheduler
+ */
+export class Schedule {
```

Comment:
Could you place the class and the related interfaces in a separate `schedule.ts` file inside the `aws-stepfunctions-tasks/lib` folder?

Reply:
I've addressed it!